llvm/lib/Target/PowerPC/PPCSubtarget.cpp

```diff
@@ -84,6 +85,8 @@ void PPCSubtarget::initSubtargetFeatures(StringRef CPU, StringRef TuneCPU,
       CPUName = "ppc64le";
     else if (TargetTriple.getSubArch() == Triple::PPCSubArch_spe)
       CPUName = "e500";
+    else if (TargetTriple.isOSAIX() && CPUName.empty())
+      CPUName = std::string(PPC::getNormalizedPPCTargetCPU(TargetTriple));
```

Comment:
For triple `ppc64le`, `getNormalizedPPCTargetCPU()` returns expected CPUName, `ppc64le`.

Will there be any issues if we just use: (We should use same CPU for clang and llc on all targets. )
```
if (CPUName.empty() || CPU == "generic") {
  if (TargetTriple.getSubArch() == Triple::PPCSubArch_spe)
    CPUName = "e500";
  else
    CPUName = std::string(PPC::getNormalizedPPCTargetCPU(TargetTriple));
}
```